Create edenlab.svg

Anykry opened the pull request:
Added Edenlab logo for powered-by page

mjsax commented:
@Anykry -- not sure if I understand this PR. The logo won't show up anywhere without a powered-by entry the html?

mjsax commented:
Cf https://github.com/apache/kafka-site/pull/503#discussion_r1645236142

Maybe we need to close this PR?

Anykry commented:
> @Anykry -- not sure if I understand this PR. The logo won't show up anywhere without a powered-by entry the html?

I've added the logo and modified powered-by.html in different PRs. And, unfortunately I made a typo in the domain name. I will fix it.
